frontend/src/scenes/data-warehouse/new/sourceWizardLogic.tsx

```diff
@@ -493,7 +498,7 @@ export const sourceWizardLogic = kea<sourceWizardLogicType>([
             },
         ],
     }),
-    selectors({
```

Comment:
@mariusandra kea logictypes are yelling at me here. Mind taking a look? It will be the latest test failure on frontend type checks